### PR TITLE
CLOUDP-73533: Fixed description iam team user add command

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-73533

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

Fixed the following bugs related to the iam team user add command:

- Added the correct description 

- Updated the Use description to specify that the commands takes a list of users' id as input

### CODE REVIEW

File: internal/cli/iam/teams/users/add.go [hunk 1]
```diff
@@ -48,13 +48,13 @@ func (opts *AddOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mongocli iam team(s) user(s) add username1 username2 .. usernameN --teamId teamId --orgId orgId
+// mongocli iam team(s) user(s) add userID1 userID2 .. userIDn --teamId teamId --orgId orgId
 func AddBuilder() *cobra.Command {
 	opts := &AddOpts{}
 	cmd := &cobra.Command{
-		Use:   "add username1 username2 .. usernameN",
+		Use:   "add userID1 userID2 .. userIDn",
```
Comment (gssbzn) [inline internal/cli/iam/teams/users/add.go:55]: let's follow [POSIX style guide
](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
> Ellipses ( "..." ) are used to denote that one or more occurrences of an operand are allowed. When an option or an operand followed by ellipses is enclosed in brackets, zero or more options or operands can be specified. The form:
>utility_name [-g option_argument]...[operand...]
indicates that multiple occurrences of the option and its option-argument preceding the ellipses are valid, with semantics as indicated in the OPTIONS section of the utility. (See also Guideline 11 in Utility Syntax Guidelines .)
>The form:
>utility_name -f option_argument [-f option_argument]... [operand...]
indicates that the -f option is required to appear at least once and may appear multiple times.

This means this should be 
```suggestion
		Use:   "add <userId> [userId]...",
```

**Review by colm-quinn — DISMISSED** [review]
LGTM

**Review by gssbzn — CHANGES_REQUESTED** [review]
thanks for fixing this one small comment to make this follow the POSIX standard

**Review by gssbzn — APPROVED** [review]
LGTM

File: internal/cli/iam/teams/users/add.go (hunk 1 above)
Comment (colm-quinn) [reply internal/cli/iam/teams/users/add.go]: TIL!

**Pull request merged**